pkgs/tools/audio/openai-whisper-cpp/default.nix

```diff
@@ -12,6 +12,7 @@
 , MetalKit
 
 , config
+, autoAddDriverRunpath
```

Comment:
Isn't L49 going to pick up the `lib.warn` alias from `cudaPackages` once we actually deprecate it?

Reply:
it is, but there's no reason for people directly consuming nixpkgs to see it.

Reply:
Shouldn't we move `[  autoAddDriverRunpath ]` out of the `with cudaPackages;` scope?

Reply:
I guess from a user's perspective, they will get eval warning from https://github.com/NixOS/nixpkgs/pull/300321